openlibrary/plugins/openlibrary/api.py

```diff
@@ -487,3 +488,34 @@ def response(msg, status="success"):
             }), content_type="application/json")
 
         return response('Observations removed')
+
+
+class works(delegate.page):
+    path = r"/works/(OL\d+W)"
+    encoding = "json"
```

Comment:
Ahhh, ok! Then let's just drop this :P It makes sense to me that called DELETE on the path itself should cause the thing to be deleted.